packages/api-v2/src/utils/display-money.ts

```diff
@@ -12,9 +12,15 @@ export const displayMoney = (
     }
 ) => {
     try {
+        const fractional_digits = Math.max(
+            // whichever is bigger:
+            options?.fractional_digits || 2, // currency's number of decimal places
+            [...amount.toString()].reverse().indexOf('.') // amount's decimal places
+        );
+
         return `${Intl.NumberFormat(options?.preferred_language || 'en-US', {
-            minimumFractionDigits: options?.fractional_digits || 2,
-            maximumFractionDigits: options?.fractional_digits || 2,
+            minimumFractionDigits: fractional_digits,
+            maximumFractionDigits: fractional_digits,
             minimumIntegerDigits: 1,
         }).format(amount)} ${currency}`;
     } catch (error) {
```

Comment:
just wondering if we can replace this implementation with the one we have in `@deriv-com/utils`. 
here is the link to the docs: https://deriv-utils.pages.dev/docs/Utils/format

Reply:
Yup good point, I've replaced it